content/en/docs/concepts/architecture/nodes.md

```diff
@@ -80,53 +80,53 @@ For self-registration, the kubelet is started with the following options:
 
     No-op if `register-node` is false.
   - `--node-ip` - IP address of the node.
-  - `--node-labels` - {{< glossary_tooltip text="Labels" term_id="label" >}} to add when registering the node in the cluster (see label restrictions enforced by the [NodeRestriction admission plugin](/docs/reference/access-authn-authz/admission-controllers/#noderestriction)).
+  - `--node-labels` - {{< glossary_tooltip text="Labels" term_id="label" >}} to add when registering the node in the cluster (see label restrictions enforced by the [`NodeRestriction` admission plugin](/docs/reference/access-authn-authz/admission-controllers/#noderestriction)).
   - `--node-status-update-frequency` - Specifies how often kubelet posts node status to master.
 
-When the [Node authorization mode](/docs/reference/access-authn-authz/node/) and
-[NodeRestriction admission plugin](/docs/reference/access-authn-authz/admission-controllers/#noderestriction) are enabled,
-kubelets are only authorized to create/modify their own Node resource.
+When the [node authorization mode](/docs/reference/access-authn-authz/node/) and
+[`NodeRestriction` admission plugin](/docs/reference/access-authn-authz/admission-controllers/#noderestriction) are enabled,
+kubelets are only authorized to create/modify their own `Node` resource.
 
-### Manual Node administration
+### Manual node administration
 
-You can create and modify Node objects using
+You can create and modify `Node` objects using
 {{< glossary_tooltip text="kubectl" term_id="kubectl" >}}.
 
-When you want to create Node objects manually, set the kubelet flag `--register-node=false`.
+When you want to create `Node` objects manually, set the kubelet flag `--register-node=false`.
 
-You can modify Node objects regardless of the setting of `--register-node`.
-For example, you can set labels on an existing Node, or mark it unschedulable.
+You can modify `Node` objects regardless of the setting of `--register-node`.
+For example, you can set labels on an existing `Node`, or mark it unschedulable.
 
-You can use labels on Nodes in conjunction with node selectors on Pods to control
-scheduling. For example, you can constrain a Pod to only be eligible to run on
+You can use labels on nodes in conjunction with node selectors on pods to control
```

Comment:
Not sure about this modification.

Reply:
I agree; the original text is referring to the objects, so I'd write:
```suggestion
You can use labels on `Nodes` in conjunction with node selectors on `Pods` to influence
```